src/setup-docker/templates/fabric-docker/commands-generated/chaincode-install-v2.sh

```diff
@@ -8,11 +8,19 @@
    - global
 */-%>
 printHeadline "Packaging chaincode '<%= chaincode.name %>'" "U1F60E"
+<% if (chaincode.directory) { -%>
 chaincodeBuild <% -%>
   "<%= chaincode.name %>" <% -%>
   "<%= chaincode.lang %>" <% -%>
   "$CHAINCODES_BASE_DIR/<%= chaincode.directory %>" <% -%>
   "<%= global.fabricRecommendedNodeVersion %>"
+<% } else if (chaincode.image) { -%>
+chaincodeBuild <% -%>
+  "<%= chaincode.name %>" <% -%>
+  "<%= chaincode.lang %>" <% -%>
+  "<%= chaincode.image %>"
+  "<%= global.fabricRecommendedNodeVersion %>"
+<% } -%>
```

Comment:
As I've written below, having two independent functions `chaincodeBuildDirPath`, `chaincodeBuildImage` would simplify this logic, thanks to it we don't need to have if statements inside functions.

Reply:
Also, to be considered - maybe it is even possible to do build->package->install if we have directory, and just a single step if we have docker image. The flow might be slightly different

Reply:
The flow for deploying Chaincode-as-a-Service is build/pull a Docker Image -> Package A Chaincode (only contains connection details) -> Install -> Run the Chaincode-as-a-service Containers.